### ISSUE TITLE
Support WebIDL partial namespaces

### ISSUE THREAD

**Issue by fitzgen** [rustwasm/wasm-bindgen#254]
Blockers:

* [ ] #253 

https://heycam.github.io/webidl/#dfn-partial-namespace

```webidl
namespace SomeNamespace {
  /* namespace_members... */
};

partial namespace SomeNamespace {
  /* namespace_members... */
};
```

Should be able to extend the AST we created for the original namespace in place.

**Comment by derekdreery** [comment]
Implemented by #678

**Comment by alexcrichton** [comment]
👍 

Thanks @derekdreery!
